scenes/game/game.gd

```diff
@@ -391,7 +401,8 @@ func begin_remote_game(game_start_message):
 		if game_start_message['starting_player_id'] == game_start_message['player1_id']:
 			starting_player = Enums.PlayerId.PlayerId_Opponent
 
-	game_wrapper.initialize_remote_game(my_player_info, opponent_player_info, starting_player, seed_value, observer_mode, starting_message_queue)
+	game_wrapper.initialize_remote_game(my_player_info, opponent_player_info, 
+		starting_player, seed_value, observer_mode, starting_message_queue,)
```

Comment:
extra comma?

Reply:
I think generally i would prefer multiline parameters to have each param on a separate line.
I make no claims about following the consistency of this rule in this project, haha.